.npmignore

```diff
@@ -21,4 +21,10 @@
 !copyright-notice.html
 !index.html
 !openmct.js
-!SECURITY.md
\ No newline at end of file
+!SECURITY.md
+
+# Add e2e tests to npm package
+!/e2e/
```

Comment:
I think this needs to be `!/e2e/**/*` instead, right now only `e2e/README.md` is being packaged